sys-apps/outils/outils-0.9.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="openbsd utils"
+HOMEPAGE="https://openbsd.org/
+	https://github.com/leahneukirchen/outils"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/leahneukirchen/outils.git"
+else
+	SRC_URI="https://github.com/leahneukirchen/outils/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86"
+fi
+
+LICENSE="BSD BSD-1 ISC"
+SLOT="0"
+
+#RDEPEND="
```

Comment:
I left this here for now for an idea of how many packages are being blocked if I don't add a prefix to the binaries.